mautic/mautic
# Set a email batch limit

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [no]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Issue(s) addressed                     | Fixes #... 



#### Description:
As a system admin I would like to not overwhelm my mail server with too many requests. Therefore, we introduce a new parameter that allows the setting of a specific rate limit.

A new setting is available to set the default batch limit for immediate processing

![image](https://user-images.githubusercontent.com/96085911/158836957-5b3722f3-1740-4a95-91f6-05c3b3d1eb3e.png)



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Email Send Settings
3. Set a banch limit
4. Save settings
5. Create a segment email
6. Send the email 
7. There is a banch limit 




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10967"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
